components/LanguageSelector.tsx

```diff
@@ -1,50 +1,55 @@
 import React from 'react';
-import {SUPPORTED_LANGUAGES} from '../i18n';
+import i18n, {SUPPORTED_LANGUAGES} from '../i18n';
```

Comment:
There are two instances is `i18n` now. One from import and another one at line no 12 from useTranslation()

Reply:
removed that additonal i18n instance